packages/dapp/src/App.tsx

```diff
@@ -42,18 +44,25 @@ export class App extends React.Component {
         <ApolloProvider client={client}>
           <ConnectedRouter history={history}>
             <CivilContext.Provider value={this.civilContext}>
-              <LoadUser>
-                {({ user: civilUser }) => {
-                  return (
-                    <>
-                      <Web3AuthWrapper civilUser={civilUser} />
-                      <GlobalNav civilUser={civilUser} />
-                      <Main civilUser={civilUser} />
-                      <Footer />
-                    </>
-                  );
-                }}
-              </LoadUser>
+              <Switch>
+                {standaloneRoutes.map(route => (
+                  <Route key={route.pathname} path={route.pathname} component={route.component} />
+                ))}
+                <Route>
+                  <LoadUser>
```

Comment:
I think the boost entry point should be inside LoadUser, we're gonna want to have user session data eventually

Reply:
FWIW `civilUser` is moving into civil context so this component won't be needed after the Kirby PR

Reply:
Is that this PR? https://github.com/joincivil/Civil/pull/1401 if so I'll leave it where it is for now.

Reply:
yep
https://github.com/joincivil/Civil/pull/1401/files#diff-027941cdab4a902d3a0dcb788bebf089R18